mwclient/client.py

```diff
@@ -95,14 +97,13 @@ def __init__(self, host, path='/w/', ext='.php', pool=None, retry_timeout=30,
             if client_certificate:
                 self.connection.cert = client_certificate
 
-            prefix = '{} - '.format(clients_useragent) if clients_useragent else ''
-            self.connection.headers['User-Agent'] = (
-                '{prefix}MwClient/{ver} ({url})'.format(
-                    prefix=prefix,
-                    ver=__ver__,
-                    url='https://github.com/mwclient/mwclient'
-                )
-            )
+            # Set User-Agent header field
+            if clients_useragent:
+                ua = clients_useragent + ' ' + USER_AGENT
```

Comment:
Would it make sense to change this to `                ua = clients_useragent + '. Using ' + USER_AGENT`?

Reply:
Having had a quick glance at RFC 7231, I think the answer to this is "no", so I'm merging this.